planner/core/integration_test.go

```diff
@@ -7533,3 +7533,24 @@ func TestExplainAnalyzeDMLCommit(t *testing.T) {
 	require.NoError(t, err)
 	tk.MustQuery("select * from t").Check(testkit.Rows())
 }
+
+func TestOverrideCTEInUnion(t *testing.T) {
```

Comment:
You can push the test to cmd/explain-test/t/cte

Reply:
Good! I moved this test to cmd/explaintest/t/cte :beers: Thanks!

Reply:
PTAL